Dockerfile

```
@@ -0,0 +1,25 @@
+FROM python:3.11.0a6-slim-buster as base
+
+RUN apt-get update && apt-get install -y \
+    gcc \
+    libffi-dev
+
+RUN pip3 install 'poetry==1.1.11' \
+    && poetry config virtualenvs.create false
+
+WORKDIR /todo-app
+COPY poetry.lock .
+COPY poetry.toml .
+COPY pyproject.toml .
```

You can combine these if you want
```suggestion
COPY poetry.lock poetry.toml pyproject.toml .
```